docs/references/javascript/organization/domains.mdx

```diff
@@ -72,5 +72,5 @@ function createDomain: (domainName: string) => Promise<OrganizationDomainResourc
 
 
 [org-domain-ref]: /docs/references/javascript/organization-domain
-[pag-res-ref]: /docs/references/javascript/types/paginated-response
-[enrollment-mode-ref]: /docs/organizations/verified-domains#enrollment-modes
\ No newline at end of file
+[pag-res-ref]: /docs/references/javascript/types/clerk-paginated-response
```

Comment:
Should probably also redirect here from `/docs/references/javascript/types/paginated-response`, as some may have the URL bookmarked

Reply:
got it ! its in the marketing repo redirects